feg/radius/src/modules/coadynamic/coa_dynamic_test.go

```diff
@@ -36,22 +39,16 @@ func TestCoaDynamic(t *testing.T) {
 		),
 		SecretSource: radius.StaticSecretSource(secret),
 		Addr:         fmt.Sprintf(":%d", 4799),
-		Ready:        make(chan bool, 1),
 	}
 	fmt.Print("Starting server... ")
 	go func() {
 		_ = radiusServer.ListenAndServe()
 	}()
 	defer radiusServer.Shutdown(context.Background())
-	listenSuccess := <-radiusServer.Ready // Wait for server to get ready
-	if !listenSuccess {
-		require.Fail(t, "radiusServer start error")
-		return
-	}
-	fmt.Println("Server listenning")
+	time.Sleep(10 * time.Millisecond)
```

Comment:
Would be nice to do this without a sleep

Reply:
Done.